source/openapi3/split.yaml

```diff
@@ -3865,34 +3304,77 @@ components:
               email: travis@hermanntorp.net
             links:
               update_bank_connection: http://go.split.cash/authorise_bank_connections/williams-and-sons-5/ddbee875-7344-4d74-9503-6233149fe1a0
-
     CreateOpenAgreementRequest:
       title: Create an Open Agreement (request)
       required:
         - title
         - terms
       type: object
       properties:
-        title:
-          type: string
-          description: Title of the Open Agreement (Visible to authorisers)
-          example: Subscription Plan A
-        terms:
-          $ref: '#/components/schemas/Terms'
-        metadata:
-          $ref: '#/components/schemas/Metadata'
-      example:
-        title: Subscription Plan A
-        terms:
-          per_payout:
-            min_amount: null
-            max_amount: 10000
-          per_frequency:
-            days: 7
-            max_amount: 1000000
-        metadata:
-          custom_key: Custom string
-          another_custom_key: Maybe a URL
+        title:
+          type: string
+          description: Title of the Open Agreement (Visible to authorisers)
+          example: Subscription Plan A
+        terms:
+          $ref: '#/components/schemas/Terms'
+        metadata:
+          $ref: '#/components/schemas/Metadata'
+      example:
+        title: Subscription Plan A
+        terms:
+          per_payout:
+            min_amount: null
+            max_amount: 10000
+          per_frequency:
+            days: 7
+            max_amount: 1000000
+        metadata:
+          custom_key: Custom string
+          another_custom_key: Maybe a URL
+    Terms:
+      title: Agreement terms
+      required:
+        - per_payout
+        - per_frequency
+      type: object
+      description: Terms
+      properties:
+        per_payout:
+          $ref: '#/components/schemas/PerPayout'
+        per_frequency:
+          $ref: '#/components/schemas/PerFrequency'
+    PerPayout:
+      title: Per payout terms
+      required:
+        - min_amount
+        - max_amount
+      type: object
+      properties:
+        min_amount:
+          type: integer
+          description: Minimum amount in cents a PR can be in order to be auto-approved. Specify <code>null</code> for no limit.
```

Comment:
Feel free to ignore, as maybe it's just me, but may I suggest avoiding `PR` acronyms? To me (and possibly other devs) PR means "pull request" and therefore this adds cognitive overhead to the docs.

Reply:
Agree!